bbp/modules/default.nix

```diff
@@ -781,6 +781,17 @@ let
             conflicts = conflicts-modules;
         };
 
+        snakemake = pkgs.envModuleGen rec {
+            name = "snakemake";
+            moduleFilePrefix = "nix/py36";
+            isLibrary = true;
+            description = "snakemake module generated by nix";
+            packages = [
```

Comment:
Adding `getPyModeRec` makes it not working for some reason.
Since it's an _application_, it works fine w/o it; should I strive to add `getPyModeRec` nevertheless?

Reply:
No, let's see why it's not working.

Reply:
Fair enough; if only I had any idea how to check it, I would. :)

Reply:
I will modify that. That's a very silly mistake I think. I'm pretty sure getPyModRec checks only for python2 for now.

Reply:
I guess getPyModRec fix will go to another PR, right?
In theory, it should not affect `snakemake` command anyway, as far as understand.
Can we merge this change already (if we are OK with it otherwise)?
It would be nice to have `nix/py36/snakemake` module available as soon as possible.